public/video-ui/src/actions/WorkflowActions/getStatus.js

```diff
@@ -20,7 +20,7 @@ function receiveStatus404() {
   return {
     type: 'WORKFLOW_STATUS_NOT_FOUND',
     receivedAt: Date.now(),
-    status: blankWorkflowStatusData
+    status: defaultWorkflowStatusData()
```

Comment:
I think this makes the UX confusing when an atom is not found to be tracked in Workflow as the form looks valid with a production office set.

Reply:
Fair enough. What would you call it? Technically it is no longer blank